### PR TITLE
feat: support v3 document

### LINKED ISSUE #23: handle v2/v3 document
I tried to use oa-token in oa-verify, however oa-token is only supporting v2 document, so I give up for the moment. Would be nice later to add support for v3

### PR DESCRIPTION
**Pull request by Nebulis**
@rjchow 

resolve #23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,52 +48,54 @@
     "OpenAttestation"
   ],
   "dependencies": {
-    "@govtechsg/open-attestation": "3.0.5",
-    "debug": "4.1.1",
-    "ethers": "4.0.41",
-    "lodash": "4.17.15"
+    "@govtechsg/open-attestation": "^3.1.1",
+    "debug": "^4.1.1",
+    "ethers": "^4.0.41",
```
Comment (rjchow) [inline package.json:53]: i thought we were pinning dependencies?

File: src/util/token.ts [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
-  return `0x${get(document, "signature.merkleRoot")}`;
+const isWrappedV3Document = (document: any): document is WrappedDocument<v3.OpenAttestationDocument> => {
+  return document && document.version === "open-attestation/3.0";
+};
+const isWrappedV2Document = (document: any): document is WrappedDocument<v2.OpenAttestationDocument> => {
+  return !isWrappedV3Document(document);
 };
 
-export const getIssuer = (document: WrappedDocument): Issuer => {
+export const getIssuerAddress = (
```
Comment (rjchow) [inline src/util/token.ts:16]: wonder if everything here should live in open-attestation? no need to do it right now just wondering

**Review by rjchow — APPROVED** [review]

File: src/util/token.ts (hunk 2 above)
Comment (Nebulis) [reply src/util/token.ts:16]: yeah agree might be helpful on open-attestation

**Pull request merged**